### PR TITLE
Added feature for chef_package_url for linux and windows system

### PR DESCRIPTION
**Pull request by NAshwini**
Signed-off-by: NAshwini <ashwini.nehate@msystechnologies.com>

## Description
Added feature for chef_package_url for linux and windows system

## Issue Resolved
https://github.com/chef-partners/azure-chef-extension/issues/297

### CODE REVIEW

**Comment by gsreynolds** [comment]
Looks good to me 👍 . Thanks @NAshwini!

File: ChefExtensionHandler/bin/chef-install.psm1 [hunk 1]
```diff
@@ -51,7 +51,7 @@ function Install-ChefClient {
   $completed = $false
 
   while (-not $completed) {
-    echo "Downloading Chef Client ..."
+    echo "Checking Chef Client ..."
```
Comment (Vasu1105) [inline ChefExtensionHandler/bin/chef-install.psm1:54]: We need add echo statements for each of the scenario here. We should be showing Downloading message when we are doing that process.

**Comment by NAshwini** [comment]
> Looks good to me  . Thanks @NAshwini!

Thank you.

**Comment by Vasu1105** [comment]
Looks Good @NAshwini. Thanks.

**Comment by NAshwini** [comment]
> Looks Good @NAshwini. Thanks.

Thank you.

**Pull request merged**